packages/wdio-browserstack-service/src/service.js

```diff
@@ -36,16 +38,18 @@ export default class BrowserstackService {
             this.sessionBaseUrl = 'https://api-cloud.browserstack.com/app-automate/sessions'
         }
 
+        this.scenariosThatRan = []
+
         return this._printSessionURL()
     }
 
-    afterSuite(suite) {
```

Comment:
We can't remove `afterSuite` as it is needed for other non Cucumber frameworks

Reply:
Is it still needed though? `afterSuite` was simply updating the `failures` counter to be used in the `after` hook: this PR now uses `after` hook's `result` for that

Reply:
You are right